codalab/objects/worksheet.py

```diff
@@ -31,7 +31,6 @@ def validate(self):
         and its name.
         '''
         spec_util.check_uuid(self.uuid)
-        spec_util.check_name(self.name)
```

Comment:
Why are these removed?

Reply:
These check if the name contains ASCII patterns

Reply:
I think the worksheet name should still be restricted (like an identifier).  The title can be free-form.